core/network/build.gradle.kts

```diff
@@ -88,4 +88,7 @@ dependencies {
     //stetho dependencies
     implementation("com.facebook.stetho:stetho:1.3.1")
     implementation("com.facebook.stetho:stetho-okhttp3:1.3.1")
+
+    //glide dependency
+    implementation("com.github.bumptech.glide:glide:4.15.1")
```

Comment:
Let's use coil

Reply:
Also inject in hilt https://github.com/android/nowinandroid/blob/main/core/network/src/main/kotlin/com/google/samples/apps/nowinandroid/core/network/di/NetworkModule.kt#L77